mpt/impl/imap-mailbox/cassandra/build.gradle

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+plugins {
+    id 'java-library'
+}
+
+configurations {
+    tests
+}
+
+dependencies {
+    implementation project(':james-backends-common:apache-james-backends-cassandra')
```

Comment:
this kind of dependency should be provided by `apache-james-mailbox:apache-james-mailbox-cassandra`. We should not have to repeat it here IMO.

Reply:
Thanks for the review and suggestion.
It is the same in the pom.xml, and this was migrated by gradle.
I'm not very familiar with the dependencies and how they should be done so I stayed away from that.
My goal is and was to make the build work as it is in maven.
We can always improve once the migration is done. 
I think this can be done as part of the re-organization we have planned. 
We can drop and merge modules then and also cleanup dependencies as part of the process.

One thing I think we should avoid is dragging this migration process for too long for minor things. 
Let's have a good migration checklist https://issues.apache.org/jira/browse/JAMES-3273 and decide what is a migration blocker and what is not.